paper-menu-button.html

```diff
@@ -90,11 +90,14 @@
       margin-top: 20px;
     }
 
-    paper-material {
+    iron-dropdown {
+      @apply(--paper-menu-button-dropdown);
+    }
+
+    .dropdown-content {
       border-radius: 2px;
       background-color: var(--paper-menu-button-dropdown-background, --primary-background-color);
-
-      @apply(--paper-menu-button-dropdown);
+      @apply(--paper-menu-button-content);
```

Comment:
`.dropdown-content` is in the local DOM, but the selector in the light DOM happens to be named `.dropdown-content` as well.